### DISCUSSION TITLE
production env issue

### DISCUSSION THREAD

**Discussion by geekerzhou** [unloggedio/unlogged-sdk#41] [Q&A]
Due to the modification of the project code and the introduction of unlogged jar packages, will there be any impact on the production environment?

**Comment by artpar** [comment]
Yes, Please do not deploy your application with unlogged-sdk in production. 

We are working on creating specific release so that production data can be recorded with minimal impact (and also allow developers to turn it off for certain flows to remove personal  data in logs). These releases of the sdk will be marked specifically that they are production ready.

There will also be a server component sitting in the middle and that is also work in progress as I write this comment.
